### ISSUE TITLE
[BUG] Specified LB VM size is ignored when using multiple LB VMs

### ISSUE THREAD

**Issue by tksh164** [Azure/aks-hybrid#205]
**Describe the bug**
Specified load balancer VM size in `New-AksHciLoadBalancerSetting` cmdlet is ignored when using multiple load balancer VMs.

I created a workload cluster with multiple load balancers aligned to the following document steps. However, the load balancer VM size was **Standard_A4_v2** (they had 4 vCPUs and **8 GB** RAM). The expected VM size was **Standard_K8S3_v1** (it has 4 vCPUs and **6 GB** RAM).

- [Use multiple load balancers in AKS on Azure Stack HCI and Windows Server](https://docs.microsoft.com/en-us/azure-stack/aks-hci/multiple-load-balancers)

    ```powershell
    $lbcfg = New-AksHciLoadBalancerSetting -name "haProxyLB" -LoadBalancerSku HAProxy -vmSize Standard_K8S3_v1 -loadBalancerCount 3

    New-AksHciCluster -name "holidays" -nodePoolName "thanksgiving" -nodeCount 2 -OSType linux -nodeVmSize Standard_A4_v2 -loadBalancerSettings $lbCfg
    ```

**To Reproduce**
Follow the following document steps to create a workload cluster with multiple load balancers.

- [Use multiple load balancers in AKS on Azure Stack HCI and Windows Server](https://docs.microsoft.com/en-us/azure-stack/aks-hci/multiple-load-balancers)

**Expected behavior**
The load balancer VMs was created with Standard_K8S3_v1 size. They have 4 vCPUs and 6 GB RAM.

**Screenshots**

![image](https://user-images.githubusercontent.com/1618784/184374563-8881bb1f-22dd-4ef4-8111-d38d9b651fc6.png)

**Environment (please complete the following information):**
- OS: Microsoft Azure Stack HCI
- Browser: n/a
- Version: 10.0.20348 N/A Build 20348
- AKS-HCI Version: 1.0.12.10727 (moduleVersion: 1.1.38)
- Kubernetes Version: v1.22.6
- AksHci module: 1.1.38
- Kva module: 1.0.31

**Additional context**
I believe that this bug comes from the implementation of the Kva module.

In `New-KvaClusterInternal` cmdlet implementation, `$loadBalancerVmSize` is set as `$global:defaultLoadBalancerVmSize` if not specified the `VmSize` or specified as `Default` in `$loadBalancerSettings`. Then generate YAML file using `$loadBalancerVmSize`.

```powershell
if (-not $($loadBalancerSettings.VmSize) -or $($loadBalancerSettings.VmSize) -eq 0) {
      $loadBalancerVmSize = $global:defaultLoadBalancerVmSize
}
```

But if set `$loadBalancerSettings.VmSize` as non `Default`, the YAML file generate using `$loadBalancerVmSize` that passthrough from the `loadBalancerVmSize` parameter of the `New-AksHciCluster` cmdlet. If not specified the `loadBalancerVmSize` parameter on the `New-AksHciCluster` cmdlet, the parameter value set to `$global:defaultLoadBalancerVmSize`. It is **Standard_A4_v2**. This is the bug.

Correctly, `$loadBalancerSettings.VmSize` should be set to `$loadBalancerVmSize` if set `$loadBalancerSettings.VmSize` as non `Default`.

**Collect log files**

n/a

**Comment by tksh164** [comment]
@madhanrm Is this will fix next the AksHci module release?

**Comment by tksh164** [comment]
I tried again with the current latest version (AksHci module: **1.1.43**, Kva module: **1.0.36**) then the result is the same. The specified LB's VM size was ignored.

**Comment by tksh164** [comment]
This issue is fixed at least AksHci module 1.1.67. (It fixed in earlier version, probably)
